### DISCUSSION TITLE
S3 failed to verify certificate

### DISCUSSION THREAD

**Discussion by MBeggiato** [pocketbase/pocketbase#2019] [Q&A] [answered]
I want to upload files to my S3 compliant storage, hosted on my network.
I allways get the following error:
`Failed to upload a test file. Raw error:
blob (key "pb_settings_test_95ltw/test.txt")
(code=Unknown): RequestError: send request
failed
caused by: Put
"https://10.92.50.31:10443/dev/pb_settings_test
_95ltw/test.txt": tls: failed to verify certificate:
x509: certificate signed by unknown authority.`

Can I disable verification?

**Comment by ganigeorgiev** [comment, marked as answer]
Are you using a self-signed certificate or don't have a TLS at all?

If you don't have tls, you can explicitly specify the protocol in the `Endpoint`, aka. `http://10.92.50.31:10443`.

**Reply by MBeggiato** [reply]
Had to create a new http endpoint. That solved the issue. Might still be a good idea to allow disabling the certificate check.
